packages/patternfly-4/content/design-guidelines/usage-and-behavior/charts/design.md

```diff
@@ -62,20 +64,58 @@ Stacked bar charts describe totals while allowing a degree of internal breakdown
 <!--#### React component
 [Stacked bar chart](/documentation/react/components/stackchart)-->
 
-### Donut chart
+### Donut charts
 Donut charts represent relative amounts that must add up to 100%.
 
 #### Usage
-The most common use cases for donut charts are to show progress completion as a percentage, show utilization for an object as a percentage and show the relationship of a set of values to a whole. Use donut charts to compare an individual section to the whole. As said previously, the entire donut should add up to 100% and the fill represents a percentage of that. There should be no more than six categories in total, similar to pie charts We recommend that you use the [Colors for charts](/design-guidelines/styles/chart-colors) to represent your data when thresholds are not present. Consider using a [Bar chart](#bar-chart) if you would like to compare one category to another.
+These are the most common use cases for donut charts:
+  - Showing the relationship of a set of values to a whole
+  - Showing progress completion as a percentage
+  - Showing utilization for an object as a percentage (with or without thresholds)
 
-#### Example
-![Donut chart](./img/donutchart.png)
+Donut charts can use percentages or whole numbers to compare parts to the whole. Consider using a [Bar chart](#bar-chart) if you would like to compare one category to another. Decide whether percentages or whole numbers is most compatible with your use case and make sure to be consistent throughout. We recommend displaying these values within the donut, see [Donut utilization](#donut-utilization) for styling. Don’t use more than six categories in total. We recommend that you use the [Colors for charts](/design-guidelines/styles/chart-colors) guidelines to represent your data when thresholds are not present.
```

Comment:
I think that "whole number" is not the right term here. If I recall from high school math, a whole number does not include a fraction. But the values we report can certainly have one or more decimal places. Maybe we should say "integer values?" @stacymcauliffe thoughts? I know this is a nit, but we are writing for an audience of engineers :)

Reply:
So the labels within the donut (i.e. 70 of 100) can be decimals as well as integers?

Reply:
Technically they can, although we might want to state a guideline about how many places to round to (or round to the nearest whole number). It might be worth asking around what we are doing in existing dashboards. The raw data will likely not be a whole number and it can look really ugly if there is no rounding.

Reply:
@mcarrano Added these guidelines